presto-main/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -242,7 +243,9 @@ public final void verifyComparableOrderableContract()
                     missingOperators.put(type, NOT_EQUAL);
                 }
             }
-            if (type.isOrderable()) {
+            if (type.isOrderable() && !(type instanceof EnumType)) {
+                // Enum types can be used in ORDER BY clauses for convenience,
```

Comment:
Let's remove this?

Reply:
I believe we said we did want to support enum values in ORDER BY clauses? 
(but we wanted to possibly implement ordering based on the keys as opposed to the values, and control it with a session param)

Reply:
I'd assume if we set EnumType as comparable = false, orderable = true things will just work? Do we really need special handling?

Reply:
if we set `comparable = false`, we get semantic errors when we write things like `mood.happy = mood.happy` or use `GROUP BY my_enum_column`

Reply:
Hmm, let's just keep both true then? While it doesn't make much sense to do `enum >= value` I don't really see a problem allowing it either, compare to the other option of inventing some new rules about type out of the spec.